.github/workflows/python-bindings.yml

```diff
@@ -0,0 +1,36 @@
+# This file is autogenerated by maturin v1.2.2
+# To update, run
+#
+#    maturin generate-ci github
+#
+name: Python bindings CI
+
+on:
+  push:
+    branches: [ $default-branch ]
+    paths:
+      - 'py-rattler/**/*'
+  pull_request:
+  workflow_dispatch:
+
+permissions:
+  contents: read
+
+jobs:
+  format_and_lint:
+    name: Format and Lint
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          submodules: recursive
+      - run: |
+          curl -fsSL https://pixi.sh/install.sh | bash
+          echo "/home/runner/.pixi/bin" >> $GITHUB_PATH
+      - uses: actions-rust-lang/setup-rust-toolchain@v1
+        with:
+          components: clippy, rustfmt
+      - run: |
+          cd py-rattler
+          pixi run lint
+          pixi run fmt
```

Comment:
This doesnt really check anything, it just runs the formatted. Maybe we can add a `check-fmt` that calls `black . --check`?